validator.js

```diff
@@ -1,982 +1,507 @@
+        new RegExp("^(?!-? )(?=.*\\d)" + pattern + "$");
+    }
+    function isCurrency(str, options) {
+        return assertString(str), options = merge(options, default_currency_options), currencyRegex(options).test(str);
+    }
+    function isBase64(str) {
+        return assertString(str), base64.test(str);
+    }
+    function ltrim(str, chars) {
+        assertString(str);
+        var pattern = chars ? new RegExp("^[" + chars + "]+", "g") : /^\s+/g;
+        return str.replace(pattern, "");
+    }
+    function rtrim(str, chars) {
+        assertString(str);
+        var pattern = chars ? new RegExp("[" + chars + "]+$", "g") : /\s+$/g;
+        return str.replace(pattern, "");
+    }
+    function trim(str, chars) {
+        assertString(str);
+        var pattern = chars ? new RegExp("^[" + chars + "]+|[" + chars + "]+$", "g") : /^\s+|\s+$/g;
+        return str.replace(pattern, "");
+    }
+    function escape(str) {
+        return assertString(str), str.replace(/&/g, "&amp;").replace(/"/g, "&quot;").replace(/'/g, "&#x27;").replace(/</g, "&lt;").replace(/>/g, "&gt;").replace(/\//g, "&#x2F;").replace(/\`/g, "&#96;");
+    }
+    function blacklist(str, chars) {
+        return assertString(str), str.replace(new RegExp("[" + chars + "]+", "g"), "");
+    }
+    function stripLow(str, keep_new_lines) {
+        assertString(str);
+        var chars = keep_new_lines ? "\\x00-\\x09\\x0B\\x0C\\x0E-\\x1F\\x7F" : "\\x00-\\x1F\\x7F";
+        return blacklist(str, chars);
+    }
+    function whitelist(str, chars) {
+        return assertString(str), str.replace(new RegExp("[^" + chars + "]+", "g"), "");
+    }
+    function isWhitelisted(str, chars) {
+        assertString(str);
+        for (var i = str.length - 1; i >= 0; i--) if (-1 === chars.indexOf(str[i])) return !1;
+        return !0;
+    }
+    function normalizeEmail(email, options) {
+        if (options = merge(options, default_normalize_email_options), !isEmail(email)) return !1;
+        var parts = email.split("@", 2);
+        if (parts[1] = parts[1].toLowerCase(), "gmail.com" === parts[1] || "googlemail.com" === parts[1]) {
+            if (options.remove_extension && (parts[0] = parts[0].split("+")[0]), options.remove_dots && (parts[0] = parts[0].replace(/\./g, "")), 
+            !parts[0].length) return !1;
+            parts[0] = parts[0].toLowerCase(), parts[1] = "gmail.com";
+        } else options.lowercase && (parts[0] = parts[0].toLowerCase());
+        return parts.join("@");
+    }
+    var babelHelpers = {};
+    babelHelpers["typeof"] = "function" == typeof Symbol && "symbol" == typeof Symbol.iterator ? function(obj) {
+        return typeof obj;
+    } : function(obj) {
+        return obj && "function" == typeof Symbol && obj.constructor === Symbol ? "symbol" : typeof obj;
+    };
+    for (var locale, default_fqdn_options = {
```

Comment:
This is a very strange way of declaring variables for use elsewhere :wink: is this even valid?